src/ActionMenu.tsx

```diff
@@ -44,11 +44,27 @@ const ActionMenuBase = ({
         ...itemProps,
         role: 'menuitem',
         onKeyPress: _event => {
-          onAction?.(itemProps as ItemProps)
+          if (itemProps.disabled) {
+            return
+          }
+
+          if (itemProps.onAction) {
```

Comment:
Would this branching logic (and the analogous `onClick`) be more readable if we followed the style of [`renderGroup`](https://github.com/primer/components/blob/a795d6ea386c39be48e3ebe715d2cb61f73435fc/src/ActionList/List.tsx#L129-L130) in `ActionList`?

Something like this:
```
const actionCallback = itemProps.onAction ?? onAction
actionCallback?.(itemProps as ItemProps)
```

Reply:
Fixed in https://github.com/primer/components/pull/1186/commits/052c8dddc1b21c9e2fb94e5be06b7f0c27ffd0b4